### PR TITLE
[PAY-267, C-753] Remove tipping feature flags, Fix mutuals icon

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

- Removes tipping feature flags, and legacy code.
- Fixes mutuals user-list icon from trophy to icon-following

### CODE REVIEW

File: packages/mobile/src/screens/profile-screen/ProfileHeader/ExpandedSection.tsx [hunk 1]
```diff
@@ -66,7 +72,9 @@ export const ExpandedSection = () => {
       <SocialsAndSites />
       <View style={{ flexDirection: 'row', marginVertical: spacing(2) }}>
         <ProfileTierTile />
-        {isOwner ? null : <ProfileMutualsButton />}
+        {isOwner || current_user_followee_follow_count === 0 ? null : (
```
Comment (sddioulde) [inline packages/mobile/src/screens/profile-screen/ProfileHeader/ExpandedSection.tsx:75]: nice

**Review by sddioulde — APPROVED** [review]
sweet! thx

File: packages/mobile/src/screens/profile-screen/ProfileHeader/ExpandedSection.tsx (hunk 1 above)
Comment (dylanjeffers) [reply packages/mobile/src/screens/profile-screen/ProfileHeader/ExpandedSection.tsx:75]: yeah! checked with julian on it too. 2 birds

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-pay-267-remove-tipping-feature-flags

**Pull request merged**